x/ibc/07-tendermint/update.go

```diff
@@ -51,46 +59,57 @@ func CheckValidityAndUpdateState(
 //
```

Comment:
could we update this godoc with a concrete explanation of what the checks are looking for and why we use the consensus state and not the client state

Reply:
This is already described in the godoc of the top level function `CheckValidityAndUpdateState` so I think it's fine to leave as-is here